new(cmake,ci): added support for using jemalloc allocator instead of glibc one (#3406)
falcosecurity/falco: CMakeLists.txt

include(ExternalProject)

> if(USE_JEMALLOC)

Reviewer: Seeing the comment about jemalloc and ASAN above, shouldn't there be a check done here that fails the configuration if both are enabled ? Otherwise we will likely have users that will get problematic builds.

Author: I agree, i'll just add a warning!

Author: Pushed!

include(ExternalProject)

if(USE_JEMALLOC)
	if(USE_ASAN)

> 		message(WARNING "Jemalloc and ASAN do not play well together")

Reviewer: Non blocking, what about:
```suggestion
		message(WARNING "Jemalloc and ASAN are known to have issues when combined")
```
?

Author: Better ahah
